### DISCUSSION TITLE
Powdered snow outline

### DISCUSSION THREAD

**Discussion by TomTheGod** [BedrockTweaks/Files#187] [Ideas]
I think that a blue outline (not a colour that's so noticeable it ruins the surrounding scenery) would be great for this. I fall into powdered snow all the time and this would help a bunch.

**Comment by SquatchHunter** [comment]
Leather Boots are your friend good sir
but yea, I can see this being "vanilla" enough along with Suspicious Sand/Gravel to be a valid pack
its a plus one from me :)

becomtweaks has this pack.
but also I have made my own version of this pack and once we are working on community packs, this will be a quick PR nad merge :)
